modules/gatsby/src/gatsby-node/process-nodes/process-nodes-markdown.js

```diff
@@ -50,12 +50,24 @@ module.exports.processNewMarkdownNode = function processNewMarkdownNode(
   // Update path
   let relPath = node.fields.slug;
   if (node.fileAbsolutePath) {
-    const index = node.fileAbsolutePath.indexOf('docs');
-    if (index !== -1) {
-      relPath = node.fileAbsolutePath.slice(index);
+
+    const {ocularConfig} = global;
+    // TODO(@javidhsueh): we might deprecate DOC_FOLDER soon.
```

Comment:
If it has no effect, let's just remove it. We obviously won't be breaking any web sites.